persistence/src/main/scala/hmda/persistence/institutions/InstitutionPersistence.scala

```diff
@@ -69,7 +69,7 @@ class InstitutionPersistence extends HmdaPersistentActor {
       }
 
     case GetInstitutionById(institutionId) =>
-      val institution = state.institutions.find(x => x.id.toString == institutionId).getOrElse(Institution(-1, "", Set(), CFPB, Bank, hasParent = false, Inactive))
+      val institution = state.institutions.find(x => x.id.toString == institutionId).getOrElse(Institution("", "", Set(), CFPB, Bank, hasParent = false, Inactive))
```

Comment:
When `GetInstitutionById` doesn't find anything (in the `getOrElse`), I chose to make the `id` of the empty institution an empty string (`""`). Is this what we want?